guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -446,19 +453,19 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         noDirectUpdate={true}
                         onSelect={(value: any) => {
                             this._sizeOption = value;
-                            if (this._sizeOption != (sizeOptions.length - 1)) {
+                            if (this._sizeOption != (sizeOptions.length)) {
                                 const newSize = sizeValues[this._sizeOption];
                                 this.props.globalState.workbench.resizeGuiTexture(newSize);
                             }
                             this.forceUpdate();
                         }}
-                    />
-                    {this._sizeOption == (sizeOptions.length - 1) &&
+                    />}
+                    {this._sizeOption == (sizeOptions.length) &&
                         <div className="divider">
                             <FloatLineComponent
                                 icon={canvasSizeIcon}
                                 iconLabel="Canvas Size"
-                                label="X"
+                                label="W"

```

Comment:
was it intentional to change the label here ( i assume this is the keycode used?)

Reply:
Yes. This is to match the design. W for width and H for height.